pkgs/development/interpreters/erlang/generic-builder.nix

```
@@ -113,10 +125,14 @@ stdenv.mkDerivation ({
   enableParallelBuilding = parallelBuild;
 
   postPatch = ''
-    patchShebangs make
+    substituteInPlace lib/wx/c_src/wxe_impl.cpp \
+      --replace 'temp > NULL' 'temp != NULL'
 
-    ${postPatch}
-  '' + optionalString (lib.versionOlder "25" version) ''
+    patchShebangs make
+  '' + optionalString isCross ''
+    substituteInPlace erts/emulator/Makefile.in \
+      --replace '`utils/find_cross_ycf`' '${pkgsBuildBuild.beam_minimal.interpreters.erlang}/lib/erlang/erts-*/bin/yielding_c_fun'
```

```suggestion
      --replace '`utils/find_cross_ycf`' '${pkgsBuildBuild.beam_minimal.interpreters.erlang}/lib/erlang/erts-*/bin/yielding_c_fun'
```
`--replace` is deprecated